### DISCUSSION TITLE
May I merge training model included in each other folders into one folder, despite of running training as same way in each other folders?

### DISCUSSION THREAD

**Discussion by ulphypro** [Project-MONAI/MONAILabel#1584] [Q&A]
Hello, all members!

I have a question.

May I merge training model included in each other folders into one folder, despite of running training as same way in each other folders?

For example...
A folder includes training models such as 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, etc..
B folder includes training models such as 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, etc..

I want to merge training models that A folder and B folder include into C folder(ex: 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22). 

 Is it possible to paste in other folders into one folder included training model?

**Comment by diazandr3s** [comment]
Hi @ulphypro,

> 
> May I merge training model included in each other folders into one folder, despite of running training as same way in each other folders?
> For example...
> A folder includes training models such as 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, etc..
> B folder includes training models such as 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, etc..

I'm not sure I understand your question here. Can you please help me to understand what those numbers mean?

You could place the pretrained files in a single folder. However, those are read by each model as shown in the config file: https://github.com/Project-MONAI/MONAILabel/blob/main/sample-apps/radiology/lib/configs/segmentation.py#L65-L69C10

**Reply by ulphypro** [reply]
Dear @diazandr3s 

Thank you for answering my question.

I'm training regarding 200 model.

But it occurred monailabel library issue, when I had trained 172 model.

So it is stopped training work.

I backed up existing 172 (training) model folder, and I'm training rest 28 model in new folder. Two model used same trainig method.

What I want is....

Is there no problem to run auto segmentation using 200 folder that merged 172 existing (training) model and rest 28 (new) model folder.
